Formula/uutils.rb

```diff
@@ -0,0 +1,70 @@
+class Uutils < Formula
+  desc "Cross-platform Rust rewrite of the GNU coreutils"
+  homepage "https://github.com/uutils/coreutils"
+  url "https://github.com/uutils/coreutils/archive/0.0.1.tar.gz"
+  sha256 "67b3fafd21e204cef4ffe04b055e148799523bc021a8ae08a399a9c847ce8e7f"
+  head "https://github.com/uutils/coreutils.git"
+
+  depends_on "make" => :build
+  depends_on "rust" => :build
+  depends_on "sphinx-doc" => :build
+
+  conflicts_with "coreutils", :because => "uutils and coreutils install the same binaries"
+  conflicts_with "aardvark_shell_utils", :because => "both install `realpath` binaries"
+  conflicts_with "truncate", :because => "both install `truncate` binaries"
+
+  def install
+    man1.mkpath
+    ENV["PATH"]="#{Formula["make"].opt_libexec}/gnubin:#{ENV["PATH"]}"
```

Comment:
I'm not `make` expert by any means but I believe https://github.com/uutils/coreutils/pull/1486 could make this replaceable by simple `gmake` invocation.